src/css/filter-column.scss

```diff
@@ -0,0 +1,37 @@
+.wrap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
The only way I can see using a mixin for flex stuff is by creating 2 mixins, one that has 
` display: flex;
  align-items: center;`
and one that has
` display: flex;
  justify-content: center;`
and applying it to the appropriate sections. Otherwise all the flex areas are unique. Is this what you were thinking?

Reply:
Exactly that yes

Reply:
Sounds good I'll complete that now and open another PR